src/coreclr/tools/superpmi/superpmi-shared/methodcontext.cpp

```diff
@@ -3610,7 +3610,7 @@ void MethodContext::recGetThreadLocalFieldInfo(CORINFO_FIELD_HANDLE field, uint3
 
     key = CastHandle(field);
     GetThreadLocalFieldInfo->Add(key, result);
-    DEBUG_REC(dmpGetThreadLocalFieldInfo(key, result));
+    DEBUG_REC(dmpGetGCThreadLocalFieldInfo(key, result));
```

Comment:
This is a bug; the function doesn't exist?

Reply:
hhm, wondering this doesn't fail during compilation?

Reply:
We might not normally build with DEBUG_REC defined.